style.css

```diff
@@ -1,340 +1,1338 @@
+.blog,
+.about,
+.faq {
+    margin-bottom: 80px;
+}
+
+
+
+
+
+
+/*Blog page */
+
+.blog__main {
+    display: flex;
+    max-width: 1280px;
+    margin: 0 auto;
+}
+
+.blog-intro {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    height: 375px;
+    margin: 0 auto;
+    background-color: #14131C;
+}
+
+.blog-intro::before {
+    width: 100%;
+    height: 100%;
+    content: '';
+    position: absolute;
+    background-image: url('images/bg-3.png');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    z-index: 1;
+    opacity: 20%;
+}
+
+.blog-intro::after {
+    width: 100%;
+    height: 100%;
+    content: '';
+    position: absolute;
+    background-image: url('images/bg-4.png');
+    background-size: contain;
+    background-position: bottom;
+    background-repeat: no-repeat;
+    z-index: 2;
+}
+
+.blog-intro__wrapper {
+    position: absolute;
+    padding-top: 65px;
+    z-index: 3;
+}
+
+.blog-intro__wrapper>.intro__title {
+    text-transform: none;
+    text-align: center;
+}
+
+.breadcrumbs {
+    display: flex;
+    justify-content: center;
+}
+
+.breadcrumbs__item {
+    position: relative;
+    padding: 6px 12px;
+}
+
+.breadcrumbs__item--active {
+    color: rgba(231, 238, 247, 0.66);
+}
+
+.breadcrumbs__item::after {
+    content: '';
+    position: absolute;
+    top: 12px;
+    right: -2px;
+    width: 2px;
+    height: 16px;
+    background-color: #D0D7FB2B;
+}
+
+.breadcrumbs__item:last-child:after {
+    display: none;
+}
+
+.sidebar {
+    flex-basis: 25%;
+    margin-right: 32px;
+}
+
+.sidebar__block {
+    padding: 24px;
+    background-color: rgba(29, 28, 40, 1);
+}
+
+.blog-content {
+    flex-basis: 75%;
+}
+
+.blog-content__photo-wrapper {
+    display: flex;
+    gap: 24px;
+}
+
+.blog-content p {
+    margin-bottom: 32px;
+}
+
+.blog__list,
+.blog__blockquote,
+.intro__subtitle,
+.blog-content__photo {
+    margin-bottom: 32px;
+}
+
+.blog__list--mark {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    counter-reset: tidbit-counter;
+    list-style-type: none;
+}
+
+.blog__list li {
+    padding-left: 30px;
+    position: relative;
+}
+
+.blog__list--mark  li {
+    margin-bottom: 8px;
+}
+
+.blog__list--disc li:before {
+    position: absolute;
+    content: '';
+    left: 10px;
+    top: 12px;
+    width: 5px;
+    height: 5px;
+    border-radius: 50%;
+    background-color: var(--color-text);
+}
+
+.blog__list--mark li:before {
+    position: absolute;
+    content: url('images/list-icon.svg');
+    left: 0;
+}
+
+.blog__list--mark li:after {
+    position: absolute;
+    content: counter(tidbit-counter);
+    counter-increment: tidbit-counter;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+}
+
+.blog__text--link {
+    color: #78BBFA;
+}
+
+.blog__blockquote {
+    padding: 16px;
+    background-color: #002754;
+    border-radius: 16px;
+}
+
+.blog__blockquote--text {
```

Comment:
Please move this class on the parent `<blockquote cite="https://">`
On the <p> tag according to 'best practices' should be margin only